addons/mail/static/src/models/discuss_public_view.js

```diff
@@ -30,6 +30,7 @@ registerModel({
                 await this.channel.toggleCall({ startWithVideo: true });
                 await this.threadView.callView.activateFullScreen();
             }
+            this.env.services.bus_service.updateChannels();
```

Comment:
This does not look like the best place to call it, the channel is already known before this step.